### PR TITLE
fix: remove modal auto-close and remove status log upon close

### LINKED ISSUE #37: Deploy document store workflow
- I think we could add after deploying document store the following:
The deployment could take a while. You can stay on the page or leave until it's successfully deployed. You can monitor the progress on Etherscan (link to TX)

- we hide the page directly when deployed. I would rather change the modal to show it has successfully been deployed. Ask for feedback from others as well

### LINKED ISSUE #43: Bug: status is still filled after opening deploy new instance
- deploy a new document store
- wait for the deployment to be successful
- click on deploy a new document store again

image

expect: status should be empty

### LINKED ISSUE #46: Suggestion: don't automatically close the deployment popup

### PR DESCRIPTION
**Pull request by ChenRen93**
1. Remove auto-closing after document store created.
2. Clear status log upon closing document store.
3. Added message to allow users to close modal after deploying, 
4. Added modal popup upon deploy transaction complete
5. Added transaction etherscan link into log when deploying document store

Closes #46 , Closes #43, Closes #37

### CODE REVIEW

**Review by Nebulis — APPROVED** [review]

File: src/components/store-deploy-block.tsx [hunk 1]
```diff
@@ -91,7 +96,7 @@ export const StoreDeployBlock: FunctionComponent<DocumentStoreAddressProps> = ({
           />
         </div>
         <div className="sm:flex pt-5">
-          <SecondaryButton onClick={() => setShowModal(false)} className="w-full mr-5 text-sm font-medium">
+          <SecondaryButton onClick={() => clearDeployStatus()} className="w-full mr-5 text-sm font-medium">
```
Comment (Nebulis) [inline src/components/store-deploy-block.tsx:99]: as a user, it's not clear what cancel does + the message is not really helpful (you can leave but I started to deploy so I can cancel to leave?)

please review this

File: src/components/util/deploy.tsx [hunk 2]
```diff
@@ -11,7 +11,11 @@ export const deployDocumentStore = async (
     const signer = await getSigner();
     log ? log("Wallet successfully decrypted.") : null;
     const factory = new DocumentStoreFactory(signer);
-    log ? log(`Deploying document store "${storeName}".`) : null;
+    log
```
Comment (Nebulis) [inline src/components/util/deploy.tsx:14]: I think I already said we should add the link to the TX, didn't I?

File: src/components/store-deploy-block.tsx (hunk 1 above)
Comment (ChenRen93) [reply src/components/store-deploy-block.tsx:99]: Okay, I have updated the button text from cancel to close. It is more associate that the button will close the popup

File: src/components/util/deploy.tsx (hunk 2 above)
Comment (ChenRen93) [reply src/components/util/deploy.tsx:20]: Okay, I have added the link to TX. in addition I minor changes in the message to reflect the changes at the top.

**Pull request merged**